### ISSUE TITLE
[BUG]:[PAL] Splinter Cell - Shadows do not display in HW unless Blending Accuracy is set to Full, causing slow down

### ISSUE THREAD

**Issue by PolygonSlayer** [PCSX2/pcsx2#5012]
### Describe the Bug

The shadows in Splinter Cell do not appear at all in HW mode unless you set the Blending Accuracy to Full, which causes a lot of areas to slow down a lot.

### Reproduction Steps

The first mission is a good example (Police Station), so play through the tutorial and on the first level there is a metal gate which casts moving dynamic shadows. In Hardware they do not show at all unless Blending Acc. is set to Full. Shortly later in the level there is a hallway with flames and smoke and the combination of these and the setting causes a large amount of slow down.
(48% fps on an RTX 3080 Ti).

In SW it all runs completely smooth however.

### Expected Behavior

That shadows appear correctly in Hardware mode without having to set Blending Acc. to high (labelled as debug as well).

### PCSX2 Revision

Nightly v1.7.2034

### Operating System

Windows 10 (64bit)

### CPU

i9-11900K

### GPU

RTX 3080 Ti

### GS Window Screenshots

Messing with most of the other settings here and no luck, these are the current ones:

![GS_WINDOW](https://user-images.githubusercontent.com/44800819/141364010-cc724abe-7bd8-4d2a-b227-2576952ac77b.png)

### Logs & Dumps

Captured several dumps from areas described above. Some with HW and some with SW.
[SplinterCell_ShadowIssues.zip](https://github.com/PCSX2/pcsx2/files/7523092/SplinterCell_ShadowIssues.zip)

![gs_20211111200157](https://user-images.githubusercontent.com/44800819/141364181-d8249219-0dd7-4de0-9636-6fe99de4ca8e.png)
![gs_20211111200325](https://user-images.githubusercontent.com/44800819/141364185-7cd9d583-e1e6-421e-941a-550a1fb07d66.png)
![gs_20211111200439](https://user-images.githubusercontent.com/44800819/141364187-e8d61c1a-ce15-4267-be4c-db2fd0c81814.png)

**Comment by RedDevilus** [comment]
Full blending is the accurate option, most games are fine with basic blending but it's going to tax performance either way. I would say it's not a real bug but a more sensitive game.

**Comment by PolygonSlayer** [comment]
I thought I'd report it anyway, feel free to change it.
Main reason is that the description for "Full" is that it's only really intended for Debugging.

**Comment by prafullpcsx2** [comment]
Can you redo the gsdumps using Dev 1961 or older?
https://buildbot.orphis.net/pcsx2/index.php?m=dl&rev=v1.7.0-dev-1961-g4273b6ce37&platform=windows-x86

**Comment by PolygonSlayer** [comment]
I've tested it again on the build you provided and here are the gsdumps:
[Dev1961_Test.zip](https://github.com/PCSX2/pcsx2/files/7539053/Dev1961_Test.zip
![Full_andSlowdown](https://user-images.githubusercontent.com/44800819/141791647-7390c3f9-126b-4794-a913-c3fe38101913.png)
)

**Comment by PolygonSlayer** [comment]
Hi,
Just done a re-test on this same area using the latest Vulkan build (v1.7.2218) and the performance is now 100% using identical settings apart from just switching to Vulkan. Shadows render proper as well. 👍 
I did try and do a capture, but the gsdumps are several hundred MBs now, but not sure if its needed?

I also tried to switch back to OpenGL and the performance still drops, but doesn't seem to be as bad as it was before (around 80% or so now).

**Comment by RedDevilus** [comment]
How did you make gsdumps that big? Did you hold down the button for multi-frame? Single frame is fine for the most part.

**Comment by PolygonSlayer** [comment]
I'm not sure, I tried several times by just pressing it once. It came in at around 100-300mb for some reason. Let me try again.

**Comment by PolygonSlayer** [comment]
Ok, I tried again and the lowest I get now is around 80mb. Same on OpenGL as well. Should I still upload some of them?

**Comment by RedDevilus** [comment]
Sure

**Comment by PolygonSlayer** [comment]
Actually managed to capture some down to 40mb now for openGL and 70mb for Vulkan. Not sure why this is, I'm pressing it as fast as I can haha.
[Dev_1.7.2218_Vulkan.zip](https://github.com/PCSX2/pcsx2/files/7842012/Dev_1.7.2218_Vulkan.zip)
[Dev_1.7.2218_openGL.zip](https://github.com/PCSX2/pcsx2/files/7842016/Dev_1.7.2218_openGL.zip)

**Comment by coornio** [comment]
Confirming that OGL struggles in this particular area in the NTSC version of the game, though surprisingly, with my RTX 2080, I still get more frames than Poly on basically the same settings. On average, it's at 80% to 90%, roughly 50-55 fps. Vulkan DGAF in this area at all and renders on average 240% uncapped. 

This honestly isn't the only game where VK handily beats OGL with blending performance. The biggest culprits I've noticed are smoke particles, like in NFS Carbon -- move the camera away from them and suddenly even OGL performs well enough. 

Fairly sure this particular issue however is not something unique. Further improvements to OGL *might* bring it closer to VK performance with blending, but probably not something that'd need to be addressed on a per-game basis? Just thinking out loud.

**Comment by lightningterror** [comment]
I'll close this as Vulkan will perform better with blending, opengl maybe could be improved a bit more down the line.

**Comment by lightningterror** [comment]
On latest dev build shadows will work on all renderers even on Minimum Blending Accuracy option now, improved by #5389
Enjoy.
